src/remediations/controller.fifi.js

```diff
@@ -115,10 +121,17 @@ exports.getRunDetails = errors.async(async function (req, res) {
         req.user.username
     );
 
-    if (!remediation) {
+    const rhcRuns = await fifi.getRHCRuns(req.params.playbook_run_id);
```

Comment:
If you make the db query and request to playbook-dispatcher run concurrently I think this operation will be faster.